zanata-common-api/src/main/java/org/zanata/rest/dto/resource/TextFlowTarget.java

```diff
@@ -50,6 +50,14 @@ public Person getTranslator() {
         return translator;
     }
 
+    @Deprecated
+    /**
+     * Zanata server will ignore this field if set,
```

Comment:
We never used this field at all. To simplify implementation on server, this should be deprecated.